runner.js

```diff
@@ -0,0 +1,23 @@
+/**
```

Comment:
❔ Should we put this into an `examples` directory or similar?

Reply:
done

Reply:
done